ui/embed.go

```diff
@@ -20,5 +20,5 @@ package ui
 import "embed"
 
 // DistContent contains ui distributed pages
-//go:embed dist
+// go:embed dist
```

Comment:
Why adding a space here?

Reply:
Because my vscode is wrong about this place, If this place is not well done, I will revise it back.

Reply:
Pls revert this line

Reply:
ok,I will.